tests/test_vqe_vqemcscf.py

```diff
@@ -42,31 +45,52 @@ def test_vqecasci_h2o_2e_2o():
 
 
 def test_ssvqecasci_h2o_2e_2o():
+
+    n_electron = 2
```

Comment:
While you're fixing it, could you also change n_electron to n_electrons? Since n_orbitals is plural, it feels a bit odd.